src/openapi/peridio-device-openapi.yaml

```diff
@@ -108,16 +108,16 @@ paths:
       description: |
         Returns information regarding whether an update is available or not.
 
-        If an update is available, additional information describing the update is returned including a presigned URL to acquire the update.
+        If no query string params are provided, the response will only contain the status.
       tags:
         - "Release Orchestration"
       parameters:
         - in: header
-          name: peridio-release-id
+          name: peridio-release-prn
           schema:
-            description: The UUID of the currently active release are on the device.
+            description: The PRN of the currently active release on the device.
             type: string
-            format: uuid
+            format: prn
```

Comment:
I'm unsure if this value like this works, I think rather than the format it should be the type linked to prn